netket/jax/distributed.py

```diff
@@ -176,6 +177,7 @@ def sharding_decorator(f, sharded_args_tree, reduction_op_tree=False):
         reduction_op_tree: a tuple/pyrtree of reduction_op, where for each output:
             reduction_op is e.g. jax.lax.psum if it is to be reduced, then f_wrapped returns a replicated array
             reduction op is False if it is not to be reduced, then f_wrapped returns a sharded array
+            reduction op is True if it is not an array/pytree, then it is returned as python object
```

Comment:
proposals for a better interface? maybe could use strings ('static', 'None', etc) ?

(cannot use None as it's ignored by the tree_map)